tardis/__init__.py

```diff
@@ -33,6 +33,126 @@
 logger.addHandler(console_handler)
 logging.getLogger("py.warnings").addHandler(console_handler)
 
+LOGGING_LEVELS = {
+    "NOTSET": logging.NOTSET,
+    "DEBUG": logging.DEBUG,
+    "INFO": logging.INFO,
+    "WARNING": logging.WARNING,
+    "ERROR": logging.ERROR,
+    "CRITICAL": logging.CRITICAL,
+}
+
+
+class FilterLog(object):
+    """
+    Filter Log Class for Filtering Logging Output
+    to a particular level
+
+    Parameters
+    ----------
+    log_level : logging object
+        allows to have a filter for the
+        particular log_level
+    """
+
+    def __init__(self, log_level):
+        self.log_level = log_level
+
+    def filter(self, log_record):
+        """
+        filter() allows to set the logging level for
+        all the record that are being parsed & hence remove those
+        which are not of the particular level
+
+        Parameters
+        ----------
+        log_record : logging.record
+            which the paricular record upon which the
+            filter will be applied
+
+        Returns
+        -------
+        boolean : True, if the current log_record has the
+            level that of the specified log_level
+            False, if the current log_record doesn't have the
+            same log_level as the specified one
+        """
+        return log_record.levelno == self.log_level
+
+
+# Setting up a list to store the Logging Filters set by logger.addFilter()
+list_of_filter = []
+
+
+def logging_state(log_state, tardis_config, specific):
```

Comment:
See Comment about lines 103-106

```suggestion
def logging_state(log_state, tardis_config, specific=None):
```

Reply:
Sure, I have made changes such that is taken care in the `run_tardis()` function 😄